.github/ISSUE_TEMPLATE/release.md

```diff
@@ -186,6 +186,7 @@ If there are delays downstream, at some point it may make sense to go ahead and
       - see note above about permissions to trigger a job
 - [ ] Scala Users discourse https://users.scala-lang.org
 - [ ] Tweet from [@scala_lang](https://twitter.com/scala_lang)
+  - [ ] And ask Toli to also share the same thing from https://fosstodon.org/@scala_lang
```

Comment:
I think it would be good if the people responsible for the release have the Fosstodon account and post about it themselves. Who is currently doing the releases? I'd happily share the credentials!

Reply:
It's me doing releases currently, so sure, I can do it myself if you send me the credentials. (On Slack, I guess.)

I'll update the PR accordingly.